data/json/monsters/insect_spider.json

```diff
@@ -401,62 +401,179 @@
   {
     "id": "mon_centipede_small",
     "type": "MONSTER",
-    "name": { "str": "huge centipede" },
-    "description": "A brightly colored, many-legged mutant centipede the size of a dog.",
+    "name": { "str": "centipede" },
+    "description": "A brightly colored mutant centipede the size of a small dog, skittering quickly in the underbrush.  Probably won't think you're suitable prey, but beware its venom.",
+    "symbol": "a",
+    "color": "light_green",
     "copy-from": "mon_centipede_giant",
-    "proportional": { "hp": 0.5, "speed": 1.1, "morale": 0.67, "melee_dice_sides": 0.67 },
```

Comment:
Why not do proportional (possibly with adjusted modifiers)?

Reply:
I changed enough values that proportional just became three extra lines, in effect. I can do a pass over the values that map nicely to compress some, but in general I find the multipliers harder to read.

Reply:
Point.